File: 02 Hands-on Activity 1 - ARG/CompareNumbers.java
public class CompareNumbers {

    public static void main(String[] args) {
      compareNumbers();
    }
  
    public static void compareNumbers() {
      Scanner scanner = new Scanner(System.in);
  
      System.out.print("Enter the first number: ");
      double firstNumber = scanner.nextDouble();
  
      System.out.print("Enter the second number: ");
      double secondNumber = scanner.nextDouble();
  
      if (firstNumber < secondNumber) {
        System.out.println(firstNumber + " is less than " + secondNumber + ".");
      } else if (firstNumber > secondNumber) {
        System.out.println(firstNumber + " is greater than " + secondNumber + ".");
      } else {
        System.out.println(firstNumber + " is equal to " + secondNumber + ".");
      }
    }
  }

File: 02 Hands-on Activity 1 - ARG/compare_numbers.py
def compare_numbers():
  """Compares two numbers entered by the user and displays a message."""
  # Get the first number from the user
  first_number = float(input("Enter the first number: "))

  # Get the second number from the user
  second_number = float(input("Enter the second number: "))

  # Compare the numbers and display a message
  if first_number < second_number:
    print(f"{first_number} is less than {second_number}.")
  elif first_number > second_number:
    print(f"{first_number} is greater than {second_number}.")
  else:
    print(f"{first_number} is equal to {second_number}.")

# Call the function to compare numbers
compare_numbers()